client/overview/survey/emoticons.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import strings from './strings';
```

Comment:
nitpick: separate external and internal dependencies

Reply:
changed in https://github.com/Automattic/woocommerce-payments/pull/8309/commits/10d6db8c05752f330a159e40054a89a299693b79 . thx